src/pkg/control/backup.go

```diff
@@ -0,0 +1,107 @@
+package control
+
+import (
+	"github.com/alcionai/corso/src/pkg/extensions"
+)
+
+// DefaultBackupOptions provides a Backup with the default values set.
+func DefaultBackupConfig() BackupConfig {
+	return BackupConfig{
+		FailureHandling: FailAfterRecovery,
+		Parallelism: Parallelism{
+			CollectionBuffer: 4,
+			ItemFetch:        4,
+		},
+		M365: BackupM365Config{
+			DeltaPageSize: 500,
+		},
+	}
+}
+
+// Backup is the set of options used for backup operations. Each set of options
```

Comment:
```suggestion
// BackupConfig is the set of options used for backup operations. Each set of options
```

Reply:
hah if only someone wrote (and used) a linter for something like this 😉